### DISCUSSION TITLE
rd_kafka_producev is unavailable: Variadic function is unvailable

### DISCUSSION THREAD

**Discussion by lampt2010** [edenhill/librdkafka#4412] [Q&A]
I am using Mac OS version 10.15 and installed librdkafka (2.2.0) to send message kafka .
i have used SwiftKafka , 1 library wrapp librdkafka and implement with swift 5
I actually seen **rd_kafka_produce** send data with  reponse code is 0 but i checked on server and didn't see message recieved because
 my server kafka need config headers attach producer 
 I try to edit **rd_kafka_producev** but got an error "rd_kafka_producev is unavailable: Variadic function is unvailable "
 
 who can solve this problem or know reason beacause i reference rd_kafka_producev  still in rdkafka.h has define
![image](https://github.com/confluentinc/librdkafka/assets/21303090/dc8fc8a7-fcc7-4263-a456-0a35fc45b7ad)
![image](https://github.com/confluentinc/librdkafka/assets/21303090/57a6dbcc-7252-4894-8dbf-5371accc42e5)

**Comment by rolandyoung** [comment]
You might be able to find help from Katura, the developers of the SwiftKafka package you are using to connect Swift to librdkafka. However, it does not seem as if that project has seen any development activity since 2019, so you may need to find a different language binding, perhaps https://github.com/swift-server/swift-kafka-client .
